index.html

```diff
@@ -0,0 +1,132 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>David's Portfolio - Mobile</title>
+</head>
+<body>
+    <section class="works">
+        <div class="grid-container">
+            <div class="card card1">
+                <img class="screenshot" src="./pics/homepage.png" alt="">
+                
+                <h1 class="main-heading"> Tonic </h1>
+                
+                <ul class="top-container">
+                    <li>
+                        <p class="canopy">CANOPY</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev">Back End Dev</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev year">2015</p>
+                    </li>
+                </ul>
+                <p class="paragraph">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
+                </p>
+                <ul class="language-button">
+                    <li><button type="button" class="l-button">html</button></li>
+                    <li><button type="button" class="l-button">css</button></li>
+                    <li><button type="button" class="l-button">javaScript</button></li>
+                </ul>   
+                <button type="button" class="see-project">See Project </button>          
+            </div>
+            
+            <div class="card card2">
+                <img class="screenshot" src="./pics/homepage2.png" alt="">
+                <h1 class="main-heading">
+                    Multi-Post<br>Stories
+                </h1>
+                <ul class="top-container">
+                    <li>
+                        <p class="canopy">CANOPY</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev">Back End Dev</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev year">2015</p>
+                    </li>
+                </ul>
+                <p class="paragraph">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
+                </p>
+                <ul class="language-button">
+                    <li><button type="button" class="l-button">html</button></li>
+                    <li><button type="button" class="l-button">css</button></li>
+                    <li><button type="button" class="l-button">javaScript</button></li>
+                </ul>
+                <button type="button" class="see-project">See Project</button>
```

Comment:
- [ ] Please, consider for each "See Project" button the Figma button styling: Active, Hover and Pressed (see design)
<img width="757" alt="Screen Shot 2022-02-11 at 18 18 30" src="https://user-images.githubusercontent.com/49064106/153607845-dc5d37af-4383-4dac-860a-0391b3569d11.png">

Reply:
Thanks for this awesome reminder. I have added the hover and pressed buttons where applicable throughout the project. @elmar8287 Please review. The updates have been pushed to the about me, work section repo. Please review.

Reply:
@davidobi Perfect! 👍 Please, if you are finished with all suggestions, then submit your PR for the next review. Happy coding! 🥇